### DISCUSSION TITLE
Why admin portal can not modify organizations or users?

### DISCUSSION THREAD

**Discussion by tainv-93** [dani-garcia/vaultwarden#3324] [Q&A] [answered]
### Deployment environment

* vaultwarden version:

* Install method:

* Clients used: 

* Reverse proxy and version: 

* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

**Comment by BlackDex** [comment, marked as answer]
It can't because it doesn't have the whole encryption/decryption and public/private key logic in there.
That is where the web-vault is for.

We can only do basic stuff which is unencrypted stored in the database, like changing the name of an organization for example, or delete it.
And we can change the access level of a user already member of an organization, but we can not add a user to it, since we can't generate the needed keys for the user to be able to access the organization vault.

**Reply by tainv-93** [reply]
@BlackDex So how can I add a new user to the organization in self-hosted?

**Reply by BlackDex** [reply]
Login into the web-vault.
See https://bitwarden.com/help/getting-started-organizations/
And https://bitwarden.com/help/managing-users/

**Reply by tainv-93** [reply]
@BlackDex But this guide is for My Organization and from Vault, I want to add a user to another organization from Admin Portal

**Reply by BlackDex** [reply]
You can't. You need to have access to the organization to be able to do that.
